assets/src/edit-story/components/keyboardShortcutsMenu/keyboardShortcutList.js

```diff
@@ -55,15 +77,33 @@ const shortcuts = {
       commands: [
         {
           label: __('Undo', 'web-stories'),
-          shortcut: [cmdOrCtrl, 'Z'],
+          shortcut: (
+            <kbd>
+              <kbd aria-label={cmdOrCtrl.title}>{cmdOrCtrl.symbol}</kbd>
+              <kbd>{'Z'}</kbd>
+            </kbd>
+          ),
         },
         {
           label: __('Redo', 'web-stories'),
-          shortcut: [cmdOrCtrl, SPECIAL_KEYS.SHIFT, 'Z'],
+          shortcut: (
+            <kbd>
+              <kbd aria-label={cmdOrCtrl.title}>{cmdOrCtrl.symbol}</kbd>
+              <kbd aria-label={SPECIAL_KEYS.SHIFT.title}>
+                {SPECIAL_KEYS.SHIFT.symbol}
+              </kbd>
+              <kbd>{'Z'}</kbd>
```

Comment:
Just double checking, these don't need translations because they are always the same value?

Reply:
yea @swissspidy can probably verify better than I, but my understanding is that the translations are more for when there's some english involved with the keys. In this scenario, two keys doesn't mean anything more than two keys.

Also my assumption for alphanumeric character keys is that there's no need to translate them because the keyboard shortcut functionality is tied to the `Z` key explicitly and doesn't work with anything else.